src/join.t.sol

```diff
@@ -285,6 +285,7 @@ contract GemJoinTest is DSTest {
         assertEq(vat.gem("REP", address(this)), 0);
         repJoin.join(address(this), 10 * WAD);
         assertEq(rep.balanceOf(address(repJoin)), 10 * WAD);
+        assertEq(rep.balanceOf(address(this)), 90 * WAD);
```

Comment:
Very minor thing, so feel free to change or not. The order of the checks is different before and after `exit`.

Reply:
Done in 1de6be7